doc/guides/bluetooth/overview.rst

```diff
@@ -0,0 +1,88 @@
+.. _bluetooth-overview:
+
+Overview
+########
+
+.. contents::
+    :local:
+    :depth: 2
+
+Supported Features
+******************
+
+Zephyr comes integrated with a feature-rich and highly configurable
+Bluetooth stack.
+
+* Bluetooth 5.0 compliant (ESR10)
+
+  * Bluetooth subsystem and infrastructure
```

Comment:
This point seems out of place in this list

Reply:
removed